Sources/SwiftCurrent_SwiftUI/Views/ViewControllerWrapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ViewControllerWrapper.swift
+//  
+//
+//  Created by Tyler Thompson on 8/7/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+#if (os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)) && canImport(UIKit)
+import SwiftUI
+import SwiftCurrent
+
+/// A wrapper for exposing `UIViewController`s that are `FlowRepresentable` to SwiftUI.
+@available(iOS 14.0, macOS 11, tvOS 14.0, *)
+public struct ViewControllerWrapper<F: FlowRepresentable & UIViewController>: View, UIViewControllerRepresentable, FlowRepresentable {
+    public typealias UIViewControllerType = F
+    public typealias WorkflowInput = F.WorkflowInput
+    public typealias WorkflowOutput = F.WorkflowOutput
+
+    public weak var _workflowPointer: AnyFlowRepresentable?
+
+    let args: WorkflowInput?
+    public init(with args: F.WorkflowInput) {
+        self.args = args
+    }
+
+    public init() { args = nil }
+
+    public func makeUIViewController(context: Context) -> F {
+        var vc: F = {
+            if let args = args {
+                return F._factory(F.self, with: args)
+            }
+            return F._factory(F.self)
+        }()
+        vc._workflowPointer = _workflowPointer
+        return vc
+    }
+
+    public func updateUIViewController(_ uiViewController: F, context: Context) { }
```

Comment:
This is empty because it is a protocol requirement. This is used when your SwiftUI view needs to update a view controller for some reason. However, that control is not given here. If you look at interop tutorials online you'll see this goes unused a good chunk of the time. Think if your SwiftUI had a `@Binding` property you'd use this method to update a value on a `UIViewController`.

Reply:
Thank you, I get it more now.  This seems fine.